### ISSUE TITLE
CREATE TABLE `groups` error with MySQL 8

### ISSUE THREAD

**Issue by JuWell** [nextcloud/server#16004]
### Steps to reproduce

1. run`nextcloud` in docker，and using `mysql 8.0.16`

2. my install script is :

   ```bash
   docker run \
   	--name nextcloud \
   	-d \
   	-p 10007:80 \
   	-v /home/server-platform/nextcloud/apps:/var/www/html/custom_apps \
   	-v /home/server-platform/nextcloud/config:/var/www/html/config \
   	-v /home/server-platform/nextcloud/data:/var/www/html/data \
   	-v /home/server-platform/nextcloud/theme:/var/www/html/themes \
   	-e MYSQL_HOST=192.168.163.50:10004 \
   	-e MYSQL_DATABASE=nextcloud \
   	-e MYSQL_USER=nextcloud \
   	-e MYSQL_PASSWORD=nextcloud \
   	-e NEXTCLOUD_ADMIN_USER=root \
   	-e NEXTCLOUD_ADMIN_PASSWORD=root \
   	-e REDIS_HOST=192.168.163.50 \
   	-e REDIS_HOST_PORT=10006 \
   	--link 192.168.163.50:10004:mysql \
   	--privileged \
   	-u root \
   	nextcloud:14.0.12
   ```

### Expected behaviour

working well

### Actual behaviour

get errors

### Server configuration

**Operating system**:

`centos 7`

**Web server:**

**Database:**

`mysql 8.0.16`

**PHP version:**

**Nextcloud version:** (see Nextcloud admin page)

`14.0.12`

**Updated from an older Nextcloud/ownCloud or fresh install:**

**Where did you install Nextcloud from:**

`docker`

#### Nextcloud log (data/nextcloud.log)

<details>
<summary>Nextcloud log</summary>

```
Error while trying to create admin user: An exception occurred while executing 'CREATE TABLE groups (gid VARCHAR(64) DEFAULT '' NOT NULL, PRIMARY KEY(gid)) DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_bin ENGINE = InnoDB ROW_FORMAT = compressed': SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'groups (gid VARCHAR(64) DEFAULT '' NOT NULL, PRIMARY KEY(gid)) DEFAULT CHARACTER' at line 1
```

</details>

### Note:

if this problem is existed, please close this, thx

i had found [this](https://help.nextcloud.com/t/nextclound-16-and-mysql-8-0-16/52689)，maybe is the right answer

**Comment by wiswedel** [comment]
That's interesting read. 
CC @nextcloud/server

**Comment by kesselb** [comment]
Good news :tada: Nextcloud 17 will work with MySQL 8: https://github.com/nextcloud/server/pull/15497

**Comment by J0WI** [comment]
https://github.com/nextcloud/documentation/blob/stable16/admin_manual/installation/system_requirements.rst#server says

> MySQL or MariaDB 5.5+ (recommended)

This should probably be clarified.

**Comment by kesselb** [comment]
> https://github.com/nextcloud/documentation/blob/stable16/admin_manual/installation/system_requirements.rst#server says
> 
> > MySQL or MariaDB 5.5+ (recommended)
> 
> This should probably be clarified.

Yes. A pull request is already open: https://github.com/nextcloud/documentation/pull/1509

**Comment by juliushaertl** [comment]
Fixed with #15497 for 17, documentation for older relases updated in nextcloud/documentation#1509
